README.md

```diff
@@ -1 +1,45 @@
-# bukstore
\ No newline at end of file
+# bukstore
+
+In this project, I have built a simle web app based Bookstore. I've achieved this by using the React.
+
+## Live Link
+
+# Coming soon!
+
+## Install
```

Comment:
- [ ] Mention here that users must run `npm install` in order to install the dependencies